inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/AgentImpl.java

```diff
@@ -1,16 +1,46 @@
 package rocks.inspectit.oce.core;
 
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 import rocks.inspectit.oce.bootstrap.IAgent;
+import rocks.inspectit.oce.core.rocks.inspectit.oce.core.config.ConfigurationCenter;
+import rocks.inspectit.oce.core.rocks.inspectit.oce.core.config.PropertySourcesInitializer;
+import rocks.inspectit.oce.core.rocks.inspectit.oce.core.config.SpringConfiguration;
 
+import java.lang.instrument.Instrumentation;
+
+/**
+ * Implementation for the {@link IAgent} interface.
+ * This clas sis responsible forsetting up the spring context for inspectIT.
+ *
+ * @author Jonas Kunz
+ */
+@Slf4j
 public class AgentImpl implements IAgent {
 
+    private AnnotationConfigApplicationContext ctx;
+
     @Override
-    public void start() {
-        System.out.println("Starting Agent");
+    public void start(String cmdArgs, Instrumentation instrumentation) {
+
+        log.info("Starting Agent...");
```

Comment:
I would add `inspectit-oce` to the start and showdown messages.. Just to be clear which agent is starting..